scripts/rehearse-private-pg17-rollout.mjs

```
@@ -1985,21 +2014,30 @@ ROLLBACK;`,
+        createdContainers,
+        networkCreated,
+        network,
+        directory,
+        docker,
+      });
+      process.stderr.write("rehearsal_cleanup_completed\n");
+    } catch (error) {
+      process.stderr.write(
+        `rehearsal_cleanup_failed:${safeRehearsalStageErrorCode(error) ?? redactedErrorChain(error)}\n`,
+      );
+      throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)